### DISCUSSION TITLE
Add the "source" attribute for the "BeatmapAttributeText"

### DISCUSSION THREAD

**Discussion by poketoine** [ppy/osu#27955] [Ideas] [answered]
Hi, as I was modifying my skin through the game editor, I wanted to be able to see the source of the map in my HUD, but it doesn't look like there is the attribute for it, I think this can be a great addition for more options in skinning.

![Capture d'écran 2024-04-21 131442](https://github.com/ppy/osu/assets/156585613/6685505a-6be9-4afa-90f3-37ed1fff6119)

**Comment by Joehuu** [comment, marked as answer]
This was added in https://github.com/ppy/osu/pull/27978.
